app/views/hyrax/dashboard/sidebar/_tasks.html.erb

```diff
@@ -10,3 +10,15 @@
       <span class="fa fa-user"></span> <span class="sidebar-action-text"><%= t('hyrax.admin.sidebar.users') %></span>
     <% end %>
   <% end %>
+
+  <% if can? :read, :admin_dashboard %>
+    <%= menu.nav_link(hyrax.embargoes_path) do %>
+      <span class="fa fa-flag"></span> <span class="sidebar-action-text"><%= t('hyrax.embargoes.index.manage_embargoes') %></span>
+    <% end %>
+  <% end %>
+
+  <% if can? :read, :admin_dashboard %>
```

Comment:
Since this check is the same as in line 14 could this check be skipped or was there a reason to keep them both?

Reply:
I don't think there is any reason to keep them both.  Combining these into one conditional statement makes a lot more sense.  I'll update this now.  Thank you for your feedback!